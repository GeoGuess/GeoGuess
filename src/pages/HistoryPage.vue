<template>
    <ContentPage>
        <div class="content">
            <HistoryTable />
        </div>
    </ContentPage>
</template>

<script>
import ContentPage from '@/components/page/ContentPage';
import HistoryTable from '@/components/history/HistoryTable';
export default {
    name: 'HistoryPage',
    components: {
        HistoryTable,
        ContentPage,
    },
};
</script>
