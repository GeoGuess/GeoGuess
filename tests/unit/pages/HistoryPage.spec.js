import HistoryPage from '@/pages/HistoryPage.vue';
import { createLocalVue, shallowMount } from '@vue/test-utils';
import appInit from '../testutils/appInit';
import { describe, it, expect } from 'vitest';

const args = appInit(createLocalVue());
describe('Home.vue', () => {
    it('test mounted', () => {
        const wrapper = shallowMount(HistoryPage, args);

        expect(wrapper).toMatchSnapshot();
    });
});
