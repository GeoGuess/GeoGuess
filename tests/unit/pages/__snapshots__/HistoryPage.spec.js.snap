// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue test mounted 1`] = `
<contentpage-stub>
  <div class="content">
    <historytable-stub></historytable-stub>
  </div>
</contentpage-stub>
`;
